src/app/components/BibPage/components/DefinitionField.jsx

```diff
@@ -10,8 +11,7 @@ const DefinitionField = ({ field, bibValues = [], additional = false }) => {
 
   return (
     <ul className={additional && 'additionalDetails'}>
-      {list
-        .flat()
+      {_flatten(list)
         .map((value, idx) => {
           if (!value) return null;
 
```

Comment:
May be for a different PR, but want to keep on your radar RTL handling:

> The front-end should respect language direction by looking for the presence of [the RTL prefix char](https://github.com/NYPL-discovery/discovery-store-poster/blob/1a2aebf5edb37e25b54c7cc69de2a9018965e781/lib/models/sierra-record.js#L138). For example, to determine whether a parallelTitle value is RTL:
>
> `const isRTL = parallelTitle.substring(0, 1) === '\u200F'`
>
> When found, the front-end should render the RTL value in [a block element with @dir=”rtl”](https://www.w3.org/International/questions/qa-html-dir#blockelements).

Reply:
Yeah, that's a different PR which hasn't been started yet. Have not forgotten.